src/main/resources/reference.conf

```diff
@@ -17,19 +17,17 @@ akka {
 
     serializers {
       agreement-added = "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementAddedSerializer"
-      agreement-activated = "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementActivatedSerializer"
-      agreement-suspended = "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementSuspendedSerializer"
-      agreement-deactivated = "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementDeactivatedSerializer"
+      agreement-deleted = "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementDeletedSerializer"
+      agreement-updated = "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementUpdatedSerializer"
       agreement-consumer-document-added = "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementConsumerDocumentAddedSerializer"
       agreement-consumer-document-removed = "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementConsumerDocumentRemovedSerializer"
       state = "it.pagopa.interop.agreementmanagement.model.persistence.serializer.StateSerializer"
     }
 
     serialization-identifiers {
       "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementAddedSerializer" = 100000
-      "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementActivatedSerializer" = 100002
-      "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementSuspendedSerializer" = 100003
-      "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementDeactivatedSerializer" = 100004
+      "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementDeletedSerializer" = 100001
+      "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementUpdatedSerializer" = 100002
```

Comment:
Don't we have the same issue here? `AgreementActivatedSerializer` shared the same id since a second ago.

Reply:
This is from `SerializerWithStringManifest` class: 
```
  /**
   * Completely unique value to identify this implementation of Serializer, used to optimize network traffic.
   * Values from 0 to 40 are reserved for Akka internal usage.
   */
  def identifier: Int
```
I think this value is used by Akka to have a unique identifier of the serializer in its network. 
This information is not involved in the persisted data, but I pick a new id to be more secure.